DOCS.txt

```diff
@@ -13,31 +13,21 @@ TABLE OF CONTENTS
 
 
 1. INTRODUCTION
-	Slayer is a program to manage BeOS teams and threads. It provides
+	Slayer is a program to manage Haiku teams and threads. It provides
 	basic functionality to kill, suspend and resume threads as well as
 	modifiying the priorities in simple and easy to use interface.
 	Slayer is freeware. You may use it and spread it freely.
```

Comment:
Maybe replace the last line with the more specific: " Slayer is distributed under the terms of GPLv3."